Documentation/bootstrapping-guidelines.md

```diff
@@ -0,0 +1,111 @@
+# Bootstrapping Guidelines
+
+To build .NET SDK from source requires previously source-built .NET SDK and toolset which is also referred as bootstrapping .NET SDK and toolset. Check quick [instructions](https://github.com/dotnet/installer/blob/main/README.md#build-net-from-source-source-build) to build SDK using source-build. This document provides guidance for getting or building  bootstrapping .NET SDK and toolset in different scenarios (covering step 2 and step 3 of source-build instructions).
+
+## Building New .NET Version
+
+Building New .NET Version is two stage process:  
```

Comment:
Perhaps we can clarify "New version" is "New major version"? This isn't needed when going from 7.0.101 to 7.0.102, for example.